packages/jazz-tools/src/tools/implementation/zodSchema/schemaTypes/GroupSchema.ts

```diff
@@ -0,0 +1,73 @@
+import { Group } from "../../../coValues/group.js";
+import {
+  ID,
+  SubscribeListenerOptions,
+  SubscribeRestArgs,
+} from "../../../coValues/interfaces.js";
+import {
+  Account,
+  RefsToResolve,
+  RefsToResolveStrict,
+  Resolved,
+  ResolveQuery,
+} from "../../../internal.js";
+import { CoreCoValueSchema } from "./CoValueSchema.js";
+import { coOptionalDefiner } from "../zodCo.js";
+import { CoOptionalSchema } from "./CoOptionalSchema.js";
+
+export interface CoreGroupSchema extends CoreCoValueSchema {
+  builtin: "Group";
+}
+
+export function createCoreGroupSchema(): CoreGroupSchema {
+  return {
+    collaborative: true as const,
+    builtin: "Group" as const,
+  };
+}
+
+export class GroupSchema implements CoreGroupSchema {
+  readonly collaborative = true as const;
+  readonly builtin = "Group" as const;
+
+  constructor(private coValueClass: typeof Group) {}
+
+  getCoValueClass(): typeof Group {
+    return this.coValueClass;
+  }
+
+  optional(): CoOptionalSchema<this> {
+    return coOptionalDefiner(this);
+  }
+
+  create(options?: { owner: Account } | Account) {
+    return this.coValueClass.create(options);
+  }
+
+  load<G extends Group, R extends ResolveQuery<GroupSchema>>(
+    id: ID<G>,
+    options?: {
+      loadAs?: Account;
+      resolve?: RefsToResolveStrict<Group, R>;
```

Comment:
Pretty sure this is a leftover from when Groups could contain profile/root fields, similarly to Accounts. We've since removed those fields, so now Groups reference no other CoValues.

I'd remove this parameter. @gdorsi what do you think?